src/pybind/mgr/dashboard/frontend/src/app/core/not-found/not-found.component.html

```diff
@@ -1,21 +0,0 @@
-<div class="horizontal-align vertical-align full-height">
-  <div class="text-center">
-    <h1 i18n>Sorry, we could not find what you were looking for.</h1>
-    <img class="img-fluid mx-auto rounded"
-         src="assets/1500px-Southern_Keeled_Octopus.jpg">
```

Comment:
It probably makes sense to remove the image file as well, as it's no longer referenced.